src/transformers/tokenization_bart.py

```diff
@@ -27,7 +27,7 @@
 merges_url = "https://s3.amazonaws.com/models.huggingface.co/bert/roberta-large-merges.txt"
 _all_bart_models = ["bart-large", "bart-large-mnli", "bart-large-cnn", "bart-large-xsum"]
 
-VOCAB_FILES_NAMES = {"vocab_file": "sentence.bpe.model"}
+VOCAB_FILES_NAMES = {"vocab_file": "sentencepiece.bpe.model"}
```

Comment:
can you delete this line and just do 
```python
vocab_files_names =  {"vocab_file": "sentencepiece.bpe.model"}
```
in line 47. 
I was confused as to why this didn't effect bart at all.

Thanks!